allennlp/modules/seq2seq_encoders/pytorch_seq2seq_wrapper.py

```diff
@@ -87,6 +88,7 @@ def forward(self,  # pylint: disable=arguments-differ
         if num_valid < batch_size:
             _, length, output_dim = unpacked_sequence_tensor.size()
             zeros = unpacked_sequence_tensor.data.new(batch_size - num_valid, length, output_dim).fill_(0)
+            zeros = Variable(zeros)
```

Comment:
pytorch 0.4 will be nice...

Reply:
In 0.3, you can't concat variables with tensors, so this would have broken, which is still decent.